sys/net/gnrc/pktdump/Kconfig

```
@@ -0,0 +1,22 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_GNRC_PKTDUMP
+    bool "Configure GNRC_PKTDUMP"
+    depends on MODULE_GNRC_PKTDUMP
+    help
+        Configure the GNRC_PKTDUMP using Kconfig.
+
+if KCONFIG_MODULE_GNRC_PKTDUMP
+
+config GNRC_PKTDUMP_MSG_QUEUE_SIZE_EXP
+    int "Queue size (as exponent of 2^n)"
```

I find this text misleading. It is not the queue size, but the exponent *for* the queue size:

```suggestion
    int "Exponent for the queue size (resulting in the queue size 2^n)"
```